model/src/main/kotlin/OrtResult.kt

```diff
@@ -188,17 +188,19 @@ data class OrtResult(
     /**
      * Return the dependencies of the given [id] (which can refer to a [Project] or a [Package]), up to and including a
      * depth of [maxLevel] where counting starts at 0 (for the [Project] or [Package] itself) and 1 are direct
-     * dependencies etc. A value below 0 means to not limit the depth.
+     * dependencies etc. A value below 0 means to not limit the depth. If [omitExcluded] is set to true, idenfitifers of
+     * excluded projects / packages are omitted from the result.
      */
-    fun getDependencies(id: Identifier, maxLevel: Int = -1): Set<Identifier> {
+    fun getDependencies(id: Identifier, maxLevel: Int = -1, omitExcluded: Boolean = false): Set<Identifier> {
         val dependencies = mutableSetOf<Identifier>()
+        val matcher = DependencyNavigator.MATCH_ALL.takeUnless { omitExcluded } ?: { !isExcluded(it.id) }
```

Comment:
How about introducing a `DependencyNavigator.MATCH_NON_EXCLUDED` for `{ !isExcluded(it.id) }`?

Reply:
That's not possible, because the `DependencyNavigator` does not know about the excluded state.